util/build_configs/README.md

```diff
@@ -27,6 +27,46 @@ Work-in-progress (2018-07-31)
   logs, CHPL_HOME workspaces, etc in this directory, without adding
   them to Git.
 
+  Let's say your host machine has Slurm; you want a Chapel multi-config
+  build with and without Gasnet (where Gasnet may use either of two widely-
+  supported comm layers); and you also want the option to use the native
+  Slurm launcher slurm-srun as well as Chapel's default Gasnet launcher(s).
+  Let's call your new project "slurm".
+
+  The existing setenv-example-1.bash script seems quite similar, except for
+  using Slurm.
+
+  To get started, copy setenv-example-1.bash to a new file "slurm.bash" and
```

Comment:
Personal preference: I find showing things like this with reproducible code snippets and short descriptions  easier to follow. I know I asked for a human-readable description, so I did not make my earlier request very clear.

e.g.

## Creating your own build script

  To get started, copy setenv-example-1.bash to a new file `slurm.bash`:

```
cp setenv-example-1.bash slurm.bash
```

To add a new build configuration to the script, we can modify the first `build_config.py` command to include an additional configuration, e.g. 

Let's add `slurm-srun` to this line:

``` bash
./build_configs.py -p $dry_run $verbose -s setenv-example-1.bash -l setenv-example-1.make.log \
 --comm=none,gasnet --substrate=none,$substrates  # no make target
```

which should now look like:

```bash
./build_configs.py -p $dry_run $verbose -s setenv-example-1.bash -l setenv-example-1.make.log \
 --comm=none,gasnet --substrate=none,$substrates  --launcher=UNSET,slurm-srun # no make target
```

...


(Side note: I'm not actually sure if this is correct, but this is my attempt to follow the instructions)

Reply:
That's fine. I'm OK to leave as is for now.